oraichain/assetlist.json

```diff
@@ -592,6 +592,39 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/och.svg"
       }
+    },
+    {
+      "description": "The TON native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/ton",
+          "exponent": 0,
+          "aliases": ["nanoTon"]
+        },
+        {
+          "denom": "ton",
+          "exponent": 9
+        }
+      ],
+      "base": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/ton",
+      "name": "Toncoin",
+      "display": "ton",
+      "symbol": "TON",
```

Comment:
Please add `traces` here, pointing it back to ton::nanoton, using the appropriate type (synthetic?) and provider